docs/developers/environment.mdx

```diff
@@ -125,13 +125,135 @@ processor.
 </TabItem>
 </Tabs>
 
-## Install Golang
+### Install Golang
 
-Celestia-app and celestia-node are written in [Golang](https://go.dev/) so we
-must install Golang to build and run them.
+Celestia-app and celestia-node are written in [Golang](https://go.dev/)
+so we must install Golang to build and run them.
 
-> If you are running an Arabica node, you will need to install Golang Version
-> 1.19.1 by changing `ver="1.18.2"` to `ver="1.19.1"`
+<Tabs groupId="network">
+<TabItem value="arabica" label="Arabica">
+
+<Tabs groupId="operating-systems">
+<TabItem value="amd" label="Ubuntu (AMD)">
+
+```sh
+ver="1.19.1"
+cd $HOME
+wget "https://golang.org/dl/go$ver.linux-amd64.tar.gz"
+sudo rm -rf /usr/local/go
+sudo tar -C /usr/local -xzf "go$ver.linux-amd64.tar.gz"
+rm "go$ver.linux-amd64.tar.gz"
+```
+
+</TabItem>
+<TabItem value="arm" label="Ubuntu (ARM)">
+
+```sh
+ver="1.19.1"
+cd $HOME
+wget "https://golang.org/dl/go$ver.linux-arm64.tar.gz"
+sudo rm -rf /usr/local/go
+sudo tar -C /usr/local -xzf "go$ver.linux-arm64.tar.gz"
+rm "go$ver.linux-arm64.tar.gz"
+```
+
+</TabItem>
+<TabItem value="apple" label="Mac (Apple)">
+
+```sh
+ver="1.19.1"
+cd $HOME
+wget "https://golang.org/dl/go$ver.darwin-arm64.tar.gz"
+sudo rm -rf /usr/local/go
+sudo tar -C /usr/local -xzf "go$ver.darwin-arm64.tar.gz"
+rm "go$ver.darwin-arm64.tar.gz"
+```
+
+</TabItem>
+<TabItem value="mac" label="Mac (Intel)">
+
+```sh
+ver="1.19.1"
+cd $HOME
+wget "https://golang.org/dl/go$ver.darwin-amd64.tar.gz"
+sudo rm -rf /usr/local/go
+sudo tar -C /usr/local -xzf "go$ver.darwin-amd64.tar.gz"
+rm "go$ver.darwin-amd64.tar.gz"
+```
+
+</TabItem>
+</Tabs>
+
+Now we need to add the `/usr/local/go/bin` directory to `$PATH`:
+
+<Tabs groupId="shell">
+<TabItem value="sh" label="sh">
+
+```sh
+echo "export PATH=$PATH:/usr/local/go/bin:$HOME/go/bin" >> $HOME/.bash_profile
+source $HOME/.bash_profile
+```
+
+</TabItem>
```

Comment:
[optional] Given the `sh` tab is identical to the `bash` tab, thoughts on removing it entirely so that there's one less choice for readers?

```suggestion
```

Reply:
yes, thank you. going to change it all around